terraform/cyhy_portscanner_acl_rules.tf

```diff
@@ -1,22 +1,22 @@
-# Allow ingress from anywhere via the Nessus UI and ssh ports
-resource "aws_network_acl_rule" "scanner_ingress_from_anywhere_via_nessus_and_ssh" {
+# Allow ingress from public subnet (NAT gateway) via the Nessus UI and ssh ports
```

Comment:
I think this should instead say, "Allow ingress from public subnet (bastion) via the...".  Nothing unsolicited can come in via the NAT gateway.

Also, the Nessus UI port has nothing to do with the port scanners and can be dropped here.

Reply:
@jsf9k good catch!